marlowe-runtime/test/Language/Marlowe/Runtime/Transaction/ConstraintsSpec.hs

```diff
@@ -237,9 +240,9 @@ spec = do
             where
               origAda = selectLovelace . txOutValueToValue $ txOrigValue
 
-      txBodyContent <- hedgehog $ genTxBodyContent BabbageEra
+      txBodyContent <- hedgehog $ genTxBodyContent ShelleyBasedEraBabbage
 
-      pure $ case adjustTxForMinUtxo ReferenceTxInsScriptsInlineDatumsInBabbageEra protocolTestnet (Just marloweAddress) txBodyContent of
+      pure $ case adjustTxForMinUtxo BabbageEraOnwardsBabbage protocolTestnet (Just marloweAddress) txBodyContent of
```

Comment:
Eventually we'll need to test in `Conway` and change these to `BabbageEraOnwardsConway`. Maybe we could centralize code like that as a module-level variable or something?